src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -759,7 +833,10 @@ static void Main()
 }
 " + trivial2uple;
 
-            CreateCompilationWithMscorlib(source).VerifyDiagnostics();
+            CreateCompilationWithMscorlib(source).VerifyDiagnostics(
+                // (6,9): error CS8204: PROTOTYPE This is not supported yet.
+                //         (int, int, int, int, int, int, int, int, int, int, int, int) x = (1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1);
+                Diagnostic(ErrorCode.ERR_PrototypeNotYetImplemented, "(int, int, int, int, int, int, int, int, int, int, int, int)").WithLocation(6, 9));
```

Comment:
Could you please split this test in two? One with tuple type declaration, another with tuple expression. The fact that we are getting only one error leads me to believe one of the error scenarios is not handled properly.